frontends/android/BitBoxApp/app/src/main/java/ch/shiftcrypto/bitboxapp/GoViewModel.java

```diff
@@ -180,19 +179,24 @@ public void pushNotify(String msg) {
     private GoEnvironment goEnvironment;
     private GoAPI goAPI;
 
+    public GoEnvironment getGoEnvironment() {
+        return goEnvironment;
+    }
+
+    public GoAPI getGoAPI() {
+        return goAPI;
+    }
+
     public GoViewModel(Application app) {
         super(app);
 
         this.goEnvironment = new GoEnvironment();
         this.goAPI = new GoAPI();
-
-        Goserver.serve(app.getApplicationContext().getFilesDir().getAbsolutePath(), this.goEnvironment, this.goAPI);
     }
 
     @Override
     public void onCleared() {
```

Comment:
can remove the whole function now.

Reply:
It actually complains if I remove it: `AndroidViewModel` doesn't have a default constructor apparently